arex-storage-config/src/main/java/com/arextest/config/model/vo/ConfigComparisonExclusions.java

```diff
@@ -10,10 +11,8 @@
  */
 @Data
 @NoArgsConstructor
-public class ConfigComparisonExclusionsVO {
-  private String appId;
-  private String operationId;
-  private String dependencyId;
-  private String operationType;
-  private List<String> exclusions;
+public class ConfigComparisonExclusions {
```

Comment:
improve the name of Class, the name of Class should be named in the singular form if possible.